### ISSUE TITLE
The `format` command adds extra newline

### ISSUE THREAD

**Issue by drewolson** [natefaubion/purescript-tidy#19]
The `format` command adds an extra newline at the end of the output send to stdout. The `format-in-place` command does not have this issue.

Here's a simple reproduction:

```
$ cat src/Example.purs 
module Example where

import Effect (Effect)
import Effect.Class.Console as Console

main :: Effect Unit
main = do
  Console.log "hello, world!"

$ purs-tidy format-in-place src/Example.purs 
$ cat src/Example.purs 
module Example where

import Effect (Effect)
import Effect.Class.Console as Console

main :: Effect Unit
main = do
  Console.log "hello, world!"

$ purs-tidy format < src/Example.purs
module Example where

import Effect (Effect)
import Effect.Class.Console as Console

main :: Effect Unit
main = do
  Console.log "hello, world!"


```

This causes my editor integration to add extra newlines when saving my files.

**Comment by thomashoneyman** [comment]
The likely culprit here is the `Console.log` statement:
https://github.com/natefaubion/purescript-tidy/blob/bf6fe6c45478a16c089b5c4abc2936eb1b6267a9/bin/Main.purs#L152

**Comment by natefaubion** [comment]
Thanks for the report! This was fixed in 0.2.
